### DISCUSSION TITLE
Does this project implement functions similar to wang weighting？

### DISCUSSION THREAD

**Discussion by hyaihjq** [astra-toolbox/astra-toolbox#432] [Q&A]
![slice1](https://github.com/astra-toolbox/astra-toolbox/assets/27874026/33417023-f502-41cf-a050-796caaecc2ba)

paper “X-ray micro-CT with a displaced detector array”

**Comment by askorikov** [comment]
Hi! No, we don't have such a weighting implemented currently.

**Reply by hyaihjq** [reply]
OK

**Comment by stephoog** [comment]
Hey! You can do this with [LEAP](https://github.com/llnl/leap).
